### PR TITLE
Add benchmarks.py to run all benchmarks, add new file with all torchbench model names

### PR DESCRIPTION
**Pull request by voznesenskym**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #94146

cc @mlazos @soumith @yanboliang @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

### CODE REVIEW

File: benchmarks/dynamo/benchmarks.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    with open(filename, "r") as fh:
+        lines = fh.readlines()
+        lines = [line.rstrip() for line in lines]
+        for line in lines:
+            line_parts = line.split(" ")
+            if len(line_parts) == 1:
+                line_parts = line.split(",")
+            model_name = line_parts[0]
+            names.add(model_name)
+    return names
```
Comment (ezyang) [inline benchmarks/dynamo/benchmarks.py:20]: why is this not just

```
{row[0] for row in csv.reader(fh)}
```

File: benchmarks/dynamo/benchmarks.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            huggingface.huggingface_main()
+        elif name in TORCHBENCH_MODELS_FILE_NAME:
+            import torchbench
+
+            torchbench.torchbench_main()
+        else:
+            print(f"Illegal model name? {name}")
+            exit(-1)
+    else:
+        import timm_models
```
Comment (ezyang) [inline benchmarks/dynamo/benchmarks.py:91]: this is reverse the conventional order

**Review by ezyang — APPROVED** [review]
okey

File: benchmarks/dynamo/benchmarks.py (hunk 1 above)
Comment (voznesenskym) [reply benchmarks/dynamo/benchmarks.py:20]: Cause I didnt know about this, also, less readable?

Comment (ezyang) [reply benchmarks/dynamo/benchmarks.py:20]: > less readable

🤔

**Comment by voznesenskym** [comment]
```
+ python3 .github/scripts/trymerge.py --comment-id 1421454244 94146
Traceback (most recent call last):
Attempting merge of 307120d6d3f7fcc3f92cfd26be891d360ad6a92a
  File ".github/scripts/trymerge.py", line 1756, in main
    merge(args.pr_num, repo,
  File ".github/scripts/trymerge.py", line 1625, in merge
    if (datetime.utcnow() - pr.last_pushed_at()).days > stale_pr_days:
  File ".github/scripts/trymerge.py", line 730, in last_pushed_at
    return datetime.fromisoformat(self.last_commit()['pushedDate'][:-1])
TypeError: 'NoneType' object is not subscriptable
```

Today is not my day... CI bug? @malfet

**Comment by malfet** [comment]
> ```
> + python3 .github/scripts/trymerge.py --comment-id 1421454244 94146
> Traceback (most recent call last):
> Attempting merge of 307120d6d3f7fcc3f92cfd26be891d360ad6a92a
>   File ".github/scripts/trymerge.py", line 1756, in main
>     merge(args.pr_num, repo,
>   File ".github/scripts/trymerge.py", line 1625, in merge
>     if (datetime.utcnow() - pr.last_pushed_at()).days > stale_pr_days:
>   File ".github/scripts/trymerge.py", line 730, in last_pushed_at
>     return datetime.fromisoformat(self.last_commit()['pushedDate'][:-1])
> TypeError: 'NoneType' object is not subscriptable
> ```
> 
> Today is not my day... CI bug? @malfet

GitHub is unhappy with you... Let's see if it likes me more ;)

**Comment by voznesenskym** [comment]
This is my punishment for disagreeing with Ed's nit

**Comment by voznesenskym** [comment]
https://github.com/pytorch/pytorch/pull/91134 looks sus

**Comment by malfet** [comment]
Ok, so following graphql:
```json
query {
  repository(owner: "pytorch", name: "pytorch") {
    pullRequest(number: 94146) {
      commits(last:1) {
        nodes {
          commit {
            oid
            committedDate
            pushedDate
          }
        }
      }
    }
  }
}
```
returns
```json
{
  "data": {
    "repository": {
      "pullRequest": {
        "commits": {
          "nodes": [
            {
              "commit": {
                "oid": "307120d6d3f7fcc3f92cfd26be891d360ad6a92a",
                "committedDate": "2023-02-07T19:37:26Z",
                "pushedDate": null
              }
            }
          ]
        }
      }
    }
  }
}
```

**Comment by malfet** [comment]
> #91134 looks sus

I though the same, but no, that is not the case

**Comment by voznesenskym** [comment]
@malfet what shall we do?

**Pull request closed without merging**